README.md

```diff
@@ -129,7 +108,7 @@ To run the project, execute the following command:
 
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues page](https://github.com/FuadNabi/budget-tracker/issues).
+Feel free to check the [issues page](https://github.com/FuadNabi/hello-rails-react/issues).
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
- [ ] [OPTIONAL]
Kindly ensure that the _Acknowledgement_ section_ is updated with the appropriate entities, on _line 125_. Apparently,[Gregoire Vella](https://www.behance.net/gregoirevella)'s design was not used on this project!